src/html/index.html

```diff
@@ -140,47 +140,47 @@ <h4 class="card-title">
       ],
     },
     basicProgressive: {
-      progressive: '//bitdash-a.akamaihd.net/content/sintel/Sintel.mp4',
-      poster: '//bitdash-a.akamaihd.net/content/sintel/poster.png',
+      progressive: 'https://cdn.bitmovin.com/content/assets/sintel/Sintel.mp4',
+      poster: 'https://cdn.bitmovin.com/content/assets/sintel/poster.png',
     },
     basicMultiProgressive: {
       progressive: [
-        {url: '//bitdash-a.akamaihd.net/content/sintel/Sintel.mp4', type: 'video/mp4', label: 'quality1'},
-        {url: '//bitdash-a.akamaihd.net/content/sintel/Sintel.mp4', type: 'video/mp4', label: 'quality2'},
+        {url: 'https://cdn.bitmovin.com/content/assets/sintel/Sintel.mp4', type: 'video/mp4', label: 'quality1'},
+        {url: 'https://cdn.bitmovin.com/content/assets/sintel/Sintel.mp4', type: 'video/mp4', label: 'quality2'},
       ],
     },
     basicHls: {
-      hls: "//bitdash-a.akamaihd.net/content/sintel/hls/playlist.m3u8",
-      poster: '//bitdash-a.akamaihd.net/content/sintel/poster.png',
+      hls: "//cdn.bitmovin.com/content/assets/sintel/hls/playlist.m3u8",
+      poster: 'https://cdn.bitmovin.com/content/assets/sintel/poster.png',
     },
     basicDash: {
-      dash: 'http://bitdash-a.akamaihd.net/content/sintel/sintel.mpd'
+      dash: 'http://cdn.bitmovin.com/content/assets/sintel/sintel.mpd'
```

Comment:
Please also change existing URLs to https. **All occurences**

Reply:
Done: 4b0e0ad9